### PR TITLE
[Ryujinx.Graphics.Gpu] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670 

---

Depends on #5358

### CODE REVIEW

**Comment by TSRBerry** [comment]
When reviewing this PR could you pls thoroughly check d61461b16f152c14b0c8b4d5fcc37f53bda9a503? 
I'm unsure if these changes could break anything.

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs [hunk 1]
```diff
@@ -136,114 +138,115 @@ private enum TextureFormat : uint
             Srgb = 0x1 << 19, // Custom encoding
 
             // Combinations
-            R8Unorm                          = R8                | RUnorm | GUnorm | BUnorm | AUnorm,        // 0x2491d
```
Comment (marysaka) [inline src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:139]: Keep alignment here for readability?

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs [hunk 2]
```diff
@@ -276,88 +279,88 @@ private enum VertexAttributeFormat : uint
             Float = 0x07,
 
             // Combinations
-            R8Unorm             = (R8 << 21)           | (Unorm << 27),   // 0x13a00000
```
Comment (marysaka) [inline src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:279]: Keep alignment here for readability?

**Review by marysaka — APPROVED** [review]
Some comments, otherwise lgtm

**Comment by TSRBerry** [comment]
Should be fine now!

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs [hunk 3]
```diff
@@ -8,6 +9,8 @@ namespace Ryujinx.Graphics.Gpu.Image
     /// </summary>
     static class FormatTable
     {
+#pragma warning disable IDE0055 // Disable formatting
+        [SuppressMessage("Design", "CA1069: Enums values should not be duplicated")]
```
Comment (AcK77) [inline src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:13]: Which ones are duplicated ?

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs [hunk 4]
```diff
@@ -244,6 +247,7 @@ private enum TextureFormat : uint
             A5B5G5R1Unorm                    = A5B5G5R1          | RUnorm | GUnorm | BUnorm | AUnorm,        // 0x24913
         }
 
+        [SuppressMessage("Design", "CA1069: Enums values should not be duplicated")]
```
Comment (AcK77) [inline src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:250]: Which ones are duplicated ?

File: src/Ryujinx.Graphics.Gpu/Image/Texture.cs [hunk 5]
```diff
@@ -1313,33 +1313,19 @@ private bool CanTextureForceAnisotropy()
...
+                                           target == Target.Texture1DArray,
+                Target.Texture2D or Target.Texture2DArray => target == Target.Texture2D ||
+                                           target == Target.Texture2DArray,
+                Target.Cubemap or Target.CubemapArray => target == Target.Cubemap ||
+                                           target == Target.CubemapArray,
+                Target.Texture2DMultisample or Target.Texture2DMultisampleArray => target == Target.Texture2DMultisample ||
+                                           target == Target.Texture2DMultisampleArray,
+                Target.Texture3D => target == Target.Texture3D,
+                _ => false,
+            };
```
Comment (AcK77) [inline src/Ryujinx.Graphics.Gpu/Image/Texture.cs:1328]: Alignment is a bit strange here

File: src/Ryujinx.Graphics.Gpu/Shader/ComputeShaderCacheHashTable.cs [hunk 6]
```diff
@@ -50,7 +50,7 @@ public void Add(CachedShaderProgram program)
             out byte[] cachedGuestCode)
         {
             program = null;
-            ShaderCodeAccessor codeAccessor = new ShaderCodeAccessor(channel.MemoryManager, gpuVa);
+            ShaderCodeAccessor codeAccessor = new(channel.MemoryManager, gpuVa);
             bool hasSpecList = _cache.TryFindItem(codeAccessor, out var specList, out cachedGuestCode);
             return hasSpecList && specList.TryFindForCompute(channel, poolState, computeState, out program);
```
Comment (AcK77) [inline src/Ryujinx.Graphics.Gpu/Shader/ComputeShaderCacheHashTable.cs:55]:
```suggestion

            return hasSpecList && specList.TryFindForCompute(channel, poolState, computeState, out program);
```

File: src/Ryujinx.Graphics.Gpu/Engine/InlineToMemory/InlineToMemoryClassState.cs [hunk 7]
```diff
@@ -175,7 +177,7 @@ unsafe struct InlineToMemoryClassState
         public uint SetI2mSpareNoop02;
         public uint SetI2mSpareNoop03;
         public fixed uint Reserved200[3200];
-        public MmeShadowScratch SetMmeShadowScratch;
+        public Array256<uint> SetMmeShadowScratch;
```
[the remark this answers is not in the document]
Comment (TSRBerry) [reply src/Ryujinx.Graphics.Gpu/Engine/InlineToMemory/InlineToMemoryClassState.cs:180]: It's correct this way, I checked with gdkchan when I made the change!

File: src/Ryujinx.Graphics.Gpu/Engine/Threed/IndirectDrawType.cs [hunk 8]
```diff
@@ -1,8 +1,11 @@
+using System.Diagnostics.CodeAnalysis;
+
 namespace Ryujinx.Graphics.Gpu.Engine.Threed
 {
     /// <summary>
     /// Indirect draw type, which can be indexed or non-indexed, with or without a draw count.
     /// </summary>
+    [SuppressMessage("Design", "CA1069: Enums values should not be duplicated")]
```
[the remark this answers is not in the document]
Comment (TSRBerry) [reply src/Ryujinx.Graphics.Gpu/Engine/Threed/IndirectDrawType.cs:8]: `Indexed` and `Count` are duplicated

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs (hunk 3 above)
Comment (TSRBerry) [reply src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:13]: Uhm a lot of them, see the line with the `Types` comment.

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs (hunk 4 above)
Comment (TSRBerry) [reply src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:250]: See the values below the `Types` comment

File: src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs (hunk 3 above)
Comment (AcK77) [reply src/Ryujinx.Graphics.Gpu/Image/FormatTable.cs:13]: Fine then (same for the other comment)

**Review by AcK77 — APPROVED** [review]

**Pull request merged**